README.md

```diff
@@ -1,45 +1,54 @@
-* To extract FFF(RJPG header that can be used by `thermal_analysis.py` of this repo) and JPG files from an SEQ video, you can run the following command:
-    ```bash
-    python split_seq.py <SEQ file name>
-    ```
+## Features
 
+<p align="center"><img src="assets/images/main.png" /><br/>Main menu</p>
 
-## Features
-1. **ROI Scaling** - Draw a (freehand) Region of Interest area to scale the rest of the image with. This is useful in cases where the region of your interest is low in contrast compared to the scale of the entire image. This drawn area can be moved around to change the region
+### Spot marking, line measurement and area marking
+
+- Extract the temperature values at marked spots
+- Plot temperature values along the marked line(s)
+- Get min, max and average values of marked regions
+
+Generates a plot for line plots and a table for measurements in the marked regions.
 
-2. **Area Measurement** - Draw a rectangle, or freehand area(s), to get the *average, minimum, and maximum* temperatures of that area. These can be moved around as well.
+<p align="center"><img src="assets/images/markings.png" /><br/>Markings on the image.</p>
 
-3. **Line Tool** - Draw a line to get a plot (temp vs pixel distance) of the temperatures along the points.
+<p align="center">
+    <img src="assets/images/graph.png" width="50%"/>
+    <img src="assets/images/table.png" width ="49%"/>
+    <br/>Plots and measurements.
+</p>
 
-4. **Spot Measurement** - Draw spots(circular areas with a small radius). Similar to 'Area Measurement'
 
-5. **Change Image Parameters** - Option to change the global parameters: *Object Distance, Relative Humidity, Reflected Apparent Temperature, Atmospheric Temperature, Emissivity* of the image. The default values are obtained from the metadatawiki
+### ROI scaling
+Scale the entire image based on values in the marked region. Use to enhance low contrast areas.
 
-6. **Change Color Map** - Change the color map representation of the thermal data (Default-Jet). Options available are: *Gray* *(No false colormap)*, *Rainbow*, and *Hot*
+<p align="center"><img src="assets/images/roi.png" /><br/>ROI scaling interface.</p>
 
-7. **Invert Scaling** - Change the way the way the image is scaled between the default scaling, and the raw image.
+### Change colormap
+Change rgb colormap to one of the following options:
 
-## To-Do
+<p align="center"><img src="assets/images/cmap.png" /><br/>Change colormap.</p>
 
-* General Interface changes for easier use 
-* Draw multiple areas in the same go
-* Line visualization while drawing from the 'Draw Line' tool
-* Change free hand to polygon 
+### Emissivity scaling
+Change reflected apparent temperature and emissivity of marked region.
+
+<p align="center"><img src="assets/images/emm.png" /><br/>Emissivity scaling interface.</p>
```

Comment:
Is this image from the latest version...? It has "Temperature" and not RAT, can you check the same?

Reply:
Updated